src/main/java/com/bitquest/bitquest/InventoryEvents.java

```diff
@@ -59,17 +59,16 @@ public InventoryEvents(BitQuest plugin) {
         trades.add(new Trade(new ItemStack(Material.CHORUS_FLOWER,8),20000));
         trades.add(new Trade(new ItemStack(Material.DIAMOND,32),20000));//honestly needed more than 8
```

Comment:
Ok, I'm just waiting for slack suggestions, and most people buy 64 diamonds for 200b, so I would at least keep the price the same